modules/apps/portal-odata/portal-odata-impl/src/main/java/com/liferay/portal/odata/internal/filter/ExpressionVisitorImpl.java

```diff
@@ -563,6 +537,91 @@ private Filter _getORFilter(Filter leftFilter, Filter rightFilter) {
 		return booleanFilter;
 	}
 
+	private Optional<Object> _getResultVisitBinaryExpressionOptional(
```

Comment:
_getResultVisitBinaryExpressionOptional doesn't follow the rest of the class name patterns.

Reply:
I dont know how I can name this method. Please, help me.

Reply:
@cgoncas